### ISSUE TITLE
TypeError: read is not a function

### ISSUE THREAD

**Issue by mk-pmb** [tufan-io/fs-read-data#1]
The API example code from the readme doesn't work, because the module exports an object with one method, instead of a function:

```bash
$ nodejs
> const read = require('fs-read-data');
undefined
> const data = read('package.json');
TypeError: read is not a function
> read
{ readFile: [Function] }
```

I'd prefer the function be exported directly.

**Comment by dkebler** [comment]
@mk-pmb Agree!  This is the best/simpliest package for just reading in any config file (incluing js!)

One year later still the readme is still incorrect.  

For single export modules I export as default as well as named.  Then one is covered.

Couple easy fixes...if dev doesn't do might do a PR

**Comment by mk-pmb** [comment]
A while ago I made a package `read-data-file`, primarily in hopes npm would show me what I'm looking for each time I search for "read data file". Nowadays it finally does, somewhat. Burried below a lot of other packages, but at least on page 1 of results. That's probably as good as npm search can get.

**Comment by sramam** [comment]
Apologies - I had stopped using this package and didn't realize it was so bothersome. 
I would be happy to accept a PR.
